services/osparc-gateway-server/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.9'
+version: '3.8'
```

Comment:
TIP: if you really need to constraint to this version of `docker-compose` , please add a test in `tests/environment-setup/test_used_docker_compose.py`. It should be very easy

Reply:
I only added this change to make a failing unit test green, I guess this was a bug already since longer times which wasnt discovered.

Reply:
reverted this